tests/test_cosserat_curve.py

```diff
@@ -50,30 +49,27 @@
 from .utils import (
     compare_test_result_pytest,
     compare_vtk_pytest,
-    get_pytest_test_name,
-    testing_input,
-    testing_temp,
 )
 
 
-def load_cosserat_curve_from_file():
+def load_cosserat_curve_from_file(reference_file_directory):
     """Load the centerline coordinates from the reference files and create the
     Cosserat curve."""
     coordinates = np.loadtxt(
-        os.path.join(testing_input, "test_cosserat_curve_centerline.txt"),
+        os.path.join(reference_file_directory, "test_cosserat_curve_centerline.txt"),
```

Comment:
Do we want to passt this as a function argument or simply call the `reference_file_directory()` inside this function?